Frontend/src/pages/register/components/RegistrationEditPanel.jsx

```diff
@@ -1,27 +1,22 @@
 import { EventSelector } from '@thewca/wca-components'
-import React, { useState } from 'react'
+import React, { useContext, useState } from 'react'
 import { useParams } from 'react-router-dom'
 import { Button, TextArea } from 'semantic-ui-react'
-import { useHeldEvents } from '../../../api/helper/hooks/use_competition_info'
+import { CompetitionContext } from '../../../api/helper/context/competition_context'
 import { updateRegistration } from '../../../api/registration/patch/update_registration'
 import { setMessage } from '../../../ui/events/messages'
-import LoadingMessage from '../../../ui/messages/loadingMessage'
 import styles from './panel.module.scss'
 
 export default function RegistrationEditPanel({ registration }) {
   const [selectedEvents, setSelectedEvents] = useState(registration.event_ids)
   const [comment, setComment] = useState(registration.comment)
   const { competition_id } = useParams()
```

Comment:
Do we still need to get this from the params, or is it included in the `competitionInfo`?
Same comment applies to a bunch of other files.

Reply:
I removed it everywhere except in the first call in `Competition.jsx`